scripts/sct_deepseg_gm.py

```diff
@@ -44,6 +44,14 @@ def get_parser():
                       description="The path where the quality control generated "
                                   "content will be saved",
                       default_value=None)
+    parser.add_option(name='-qc-dataset',
+                      type_value='str',
+                      description='Name of the dataset to display in the QC report. By default, assumes BIDS.',
```

Comment:
I think there could be a clearer and more concise way to express this, some questions can arise from this wording:

- Are we displaying the dataset in the QC report?
- Are we "displaying" the name of the dataset?
- Is the string "BIDS" the default name of the dataset?

I'd use the word "mention" instead of "display". 
Since the parameter is optional, I'd use "If provided, this string will be mentioned in the QC report as the dataset the process was run on", or something like that.

Reply:
fixed in 1b7663634d88bbf00461c1b580a0c9b47b544c4e